pyxrf/xanes_maps/xanes_maps_api.py

```diff
@@ -83,8 +83,43 @@ def build_xanes_map_api(start_id=None, end_id=None, *, param_file_name,
     used if properly generated XRF maps for the selected set of element emission line is
     already present in the .h5 files.
 
-
-    Format of the reference file (see parameter ``ref_file_name``).
+    Example of function call for the batch of scans with IDs in the range 92276-92335:
+
+    build_xanes_map(92276, 92335, param_file_name="param_335", scaler_name="sclr1_ch4",
+    sequence="load_and_process", ref_file_name="refs_Fe_P23.csv", emission_line="Fe_K",
+    emission_line_alignment="P_K")
+
+    Options:
+
+    Typical processing sequence includes operation of XRF map interpolation and stack alignment.
+    But those operations may be disabled by setting parameters ``interpolation_enable`` and
+    ``alignment enable`` to False. For example, if the position coordinates of XRF maps form
+    uniform grid, then disabling interpolation will reduce processing time.
+
+    There are two parameters that control stack alignment. The stack alignment for all emission
+    lines is performed based on the emission line selected for XANES (parameter ``emission_line``).
+    If maps for different emission line are better suited for alignment, then the default behavior
+    can be changed by setting the parameter ``emission_line_alignment``. The other parameter
+    controls the order of the stack alignment. The default order (starting from the top of
+    the stack or XRF map measured with highest energy) may be changed by setting the parameter
+    ``alignment_starts_from="bottom"``. In this case the alignment will start from the map
+    acquired with the lowest incident energy.
+
+    Incident energy used for generation of XRF maps is automatically computed as the lowest of
+    the energy read from the respective .h data file or the energy which activates the
+    emission line of interest (parameter ``emission_line``). The default behavior may be
+    changed by setting True the parameter ``use_incident_energy_from_param_file`` (use fixed
+    energy from parameter file set by ``param_file_name``) or specifying lower bound for
+    incident energy ``incident_energy_low_bound`` (if the energy read from the .h5 data file
+    is less than ``incident_energy_low_bound``, then use the lower bound, otherwise use
+    the value of energy from the data file.
+
+    If ``plot_results`` is set to False, then the results will not be plotted (no Matplotlib
+    windows will be opened). The processing results will still be saved to files with selected
```

Comment:
"... files with the selected formats."

Reply:
There are multiple output files, but it may be one format (currently only TIFF). I changed it to format(s)